Update blog with post detailing events of the outage on October 6 (#3508)
pulumi/pulumi-hugo: themes/default/content/blog/post-mortem-2023-10-06/index.md

# the time portion of the date value; posts are sorted in descending order by
# date/time.
date: 2023-10-16T11:52:58-07:00

# The draft setting determines whether a post is published. Set it to true if
# you want to be able to merge the post without publishing it.
draft: false

# Use the meta_desc property to provide a brief summary (one or two sentences)
# of the content of the post, which is useful for targeting search results or
# social-media previews. This field is required or the build will fail the
# linter test. Max length is 160 characters.
meta_desc: A discussion on the Pulumi Service outage seen Friday, October 6.

# The meta_image appears in social-media previews and on the blog home page. A
# placeholder image representing the recommended format, dimensions and aspect
# ratio has been provided for you.
meta_image: meta.png

# At least one author is required. The values in this list correspond with the
# `id` properties of the team member files at /data/team/team. Create a file for
# yourself if you don't already have one.
authors:
    - casey-huang

# At least one tag is required. Lowercase, hyphen-delimited is recommended.
tags:

>     - post-mortem

Reviewer: Total nitpick:

```suggestion
    - postmortem
```

>     - outage

Reviewer: ```suggestion
```

# you want to be able to merge the post without publishing it.
draft: false

# Use the meta_desc property to provide a brief summary (one or two sentences)
# of the content of the post, which is useful for targeting search results or
# social-media previews. This field is required or the build will fail the
# linter test. Max length is 160 characters.
meta_desc: A discussion on the Pulumi Service outage seen Friday, October 6.

# The meta_image appears in social-media previews and on the blog home page. A
# placeholder image representing the recommended format, dimensions and aspect
# ratio has been provided for you.
meta_image: meta.png

# At least one author is required. The values in this list correspond with the
# `id` properties of the team member files at /data/team/team. Create a file for
# yourself if you don't already have one.
authors:
    - joe-duffy
    - casey-huang

# At least one tag is required. Lowercase, hyphen-delimited is recommended.
tags:
    - post-mortem
    - outage

# See the blogging docs at https://github.com/pulumi/pulumi-hugo/blob/master/BLOGGING.md
# for details, and please remove these comments before submitting for review.
---

> Last Friday, Pulumi suffered a 24 minute outage during which we failed to process 74.7% of received requests.

Reviewer 2: ```suggestion
At Pulumi, we understand that Pulumi Cloud plays an important role in how our customers address their infrastructure management challenges. As a result, we strive for the highest levels of availability and performance in Pulumi Cloud. Unfortunately, on Friday, October 6, 2023, Pulumi Cloud suffered a 24 minute outage during which we failed to process 74.7% of received requests. In this post, we'd like to share our findings on the root cause of this outage, and the steps we are taking to ensure this sort of outage doesn't happen again.
```

Reviewer 2: Just one potential way to rewrite the opening. Right now it feels very abrupt.
